libs/designer-ui/src/lib/workflowparameters/workflowparametersField.tsx

```diff
@@ -401,7 +401,7 @@ export const WorkflowparameterField = ({
               id={parameterDetails.value}
               ariaLabel={valueTitle}
               placeholder={valueDescription}
-              value={value}
+              value={value || ''}
```

Comment:
Why is there a need to default it to empty string? Is there something else which is not working because it works fine in workflow parameters panel.